snuba/snuba_migrations/profile_chunks/0001_create_profile_chunks_table.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime64, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set = StorageSetKey.PROFILE_CHUNKS
+table_prefix = "profile_chunks"
+local_table_name = f"{table_prefix}_local"
+dist_table_name = f"{table_prefix}_dist"
+
+UNKNOWN_SPAN_STATUS = 2
+
+columns: List[Column[Modifiers]] = [
+    Column("project_id", UInt(64)),
+    Column("profiler_id", UUID()),
+    Column("chunk_id", UUID()),
```

Comment:
In relay, these 2 fields are typed as `String` with a comment that it should be UUIDs. Should we go back and validate that to ensure non UUID values don't make it here?

Reply:
I think we'll update the fields to be proper UUIDs so I'll keep the UUID columns here.

Reply:
https://github.com/getsentry/relay/pull/3588